getting_started/overview.adoc

```diff
@@ -13,8 +13,15 @@ To get started with OpenShift, find the appropriate topic based on your role:
 |===
 |I am a... |Links to relevant topics
 
+ifdef::openshift-enterprise[]
 .^|link:administrators.html[Platform administrator]
 |link:../install_config/install/quick_install.html[Quick Install]
+endif::openshift-enterprise[]
+
+ifdef::openshift-origin[]
+.^|link:administrators.html[Platform administrator]
+|link:../getting_started/administrators.html[Quick Install]
```

Comment:
- For consistency, probably could remove the link from "Platform administrator" completely (since "Developer" below it is not a link), and just let the text in the 2nd column (under "Link to relevant topics") be the link.
- The `../getting_started/` part of the path in line 23 isn't required since you're already in that directory.
- For the link text in the 2nd column, we probably don't want to use the phrase "Quick Install" for Origin at all since we use that to refer to the installer tool available from install.openshift.com. And currently, install.openshift.com does not work for v3 Origin (it does for v2). Relatedly, we could probably remove the `quick_install` topic from the Origin build spec completely.